ford/README.md

```diff
@@ -1,3 +1,7 @@
 # Ford Mustang Mach E OBD Data
 
 These OBD PIDs are based on the work by Ben Karleskint
+
+# Ford Focus Electric 2012
+
+Based on https://revisto.eu/ 
```

Comment:
Maybe add your name here? That website does not contain much information...

Reply:
I'll update website soon (next week)